### ISSUE TITLE
Support for Slack Buttons

### ISSUE THREAD

**Issue by owocki** [scrapinghub/slackbot#90]
Per https://api.slack.com/docs/message-buttons

**Comment by phlhayes** [comment]
Is this not currently supported?

**Comment by botcippital** [comment]
No, When a button is pressed, `Slack sends a request to your Action URL`.  Action URLs are not currently supported in slackbot.

**Comment by owocki** [comment]
No, When a button is pressed, `Slack sends a request to your Action URL`.  Action URLs are not currently supported in slackbot.

**Comment by jtatum** [comment]
This would require embedding a web server and adding an oauth flow, since message buttons require you to configure a full-fledged slack app rather than just adding a bot integration.

**Comment by stack-0verfl0w** [comment]
@jtatum hey, Just wanted to confirm if there is a plan to switch to the app framework supporting the web-server, since I saw the above has been labeled as an enhancement :)
